test/no-native.js

```diff
@@ -1,27 +1,19 @@
 'use strict'
 
-var rule = require('../rules/no-native')
-var RuleTester = require('eslint').RuleTester
-var parserOptions = { sourceType: 'module', ecmaVersion: 6 }
-var ruleTester = new RuleTester()
+const rule = require('../rules/no-native')
+const RuleTester = require('eslint').RuleTester
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  }
+})
+
 ruleTester.run('no-native', rule, {
   valid: [
     'var Promise = null; function x() { return Promise.resolve("hi"); }',
     'var Promise = window.Promise || require("bluebird"); var x = Promise.reject();',
-    {
-      code:
-        'var Promise = null; function x() { return Promise.resolve("hi"); }',
-      parserOptions: parserOptions
-    },
-    {
-      code:
-        'var Promise = window.Promise || require("bluebird"); var x = Promise.reject();',
```

Comment:
This test case and the previous are duplicates of first two test cases and were removed by `npm run lint -- --fix` (per [`eslint-plugin/no-identical-tests`](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/no-identical-tests.md)).